apps/base/models.py

```diff
@@ -14,3 +19,28 @@ class TimeStampedModel(models.Model):
     class Meta:
         abstract = True
         app_label = 'base'
+
+
+"""
+The decorator below is used to pass model field names to django signals function.
+"""
+
+
+def extra_args(field_name, *args, **kwargs):
+    def extra_args_decorator(f, *args, **kwargs):
+        def extra_args_wrapper(sender, instance, **kwargs):
+            f(sender, instance, field_name=field_name, **kwargs)
+        return extra_args_wrapper
+    return extra_args_decorator
+
+
+"""
+The function below is used to log the change when any attribute of model field instance is changed.
+"""
+
+
+def create_post_model_field(sender, instance, field_name, **kwargs):
+    if getattr(instance, "_original_{}".format(field_name)) is False:
+        logger.info("{} is added first time !".format(field_name))
```

Comment:
Though this message is just an informatory message, but since we are getting information over here, so what if we log `instance.pk` in the log statements ?